### PR TITLE
Quiz end

### PR DESCRIPTION
**Pull request by vladi3ir**
this code includes the DAL and store communication for SQL, does not include Service yet

### CODE REVIEW

File: PokeQuizWebAPI/Models/QuizModels/QuizResultsViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PokeQuizWebAPI.Models.QuizModels
+{
+    public class QuizResultsViewModel
+    {
+        public string UserName { get; set; }
+        public int Score { get; set; }
```
Comment (Eddonomochi) [inline PokeQuizWebAPI/Models/QuizModels/QuizResultsViewModel.cs]: You are going to have to change this to match the new field names in tables

File: PokeQuizWebAPI/PokemonDAL/Models/PokemonDALModel.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    {
+        public string UserName { get; set; }
+        public int Score { get; set; }
+        public int QuizLength25 { get; set; }
+        public int QuizLength50 { get; set; }
+        public int QuizLength100 { get; set; }
+        public int CorrectAmount { get; set; } 
+        public float AverageScore { get; set; } 
+        public int LastScoreSession { get; set; } 
+        public string WhichQuizTaken { get; set; }
```
Comment (Eddonomochi) [inline PokeQuizWebAPI/PokemonDAL/Models/PokemonDALModel.cs]: bitmoji

These will have to be changed as well

Comment (vladi3ir) [reply PokeQuizWebAPI/PokemonDAL/Models/PokemonDALModel.cs]: Ok will work on the update sometime over the weekend

Comment (vladi3ir) [reply PokeQuizWebAPI/PokemonDAL/Models/PokemonDALModel.cs]: Updated these variable names

**Comment by Zach0702** [comment]
Merge conflict was around similar 'Using' statements and dependencies injection in the service file. I merged all those changes. 

Also in the 'Quiz View Model' , I merged the version in the master fie and removed my original view .

**Pull request merged**